### PR TITLE
Update: Alerts & Notifications Best Practices Guide

### PR DESCRIPTION
**Pull request by MimiDumpling**
Updated the Guide to be more tutorial based.
ToDo:
- [x] find more places for links
- [x] double check all links
- [x] create cleaner screenshots

### CODE REVIEW

File: src/docs/product/sentry-basics/guides/alert-notifications/issue-alerts.mdx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## Step 1: Choose an Environment
+Specify which [environment(s)](/platforms/javascript/configuration/environments/) will use this particular alert rule. `Environment` is a Sentry supported tag that you can (and should) add to your SDK. Generally, the [tag](/platforms/javascript/enriching-events/tags/) accepts any value but is targeted to refer to your code deployments' naming convention such as _Development_, _Testing_, _Staging_, or _Production_. The environment drop-down list is populated with the environment tag values available in your [Event Stream](/product/accounts/quotas/manage-event-stream-guide/).
+
+Each environment necessitates different levels of urgency and workflow. The urgency and workflows you apply to Production alerts might differ from those you apply to alerts originating from your QA environment.
+
+## Step 2: Give the Alert Rule a Name
+Manage your alert rule by giving it a descriptive name. A descriptive alert rule name specifies the team affected and the topic of the alert. For example, "Frontend Latency," "Backend Failure Rate," or "Billing Apdex."
+
+MIMI NOTE: Ask Adi for a use case
```
Comment (MimiDumpling) [inline src/docs/product/sentry-basics/guides/alert-notifications/issue-alerts.mdx:17]: @adhiraj will you provide a use case for this section?

File: src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Step 1. Choose Errors or Transactions
+
+In the Alert Conditions section, choose between [Errors](/platforms/javascript/usage/#capturing-errors) or [Transactions](/platforms/javascript/performance/capturing/). Errors provide data around a break, such as new issues or total errors in your project. Transactions provide data on the performance of your application, such as [latency](/product/performance/metrics/#latency) and [failure rate](/product/performance/metrics/#failure-rate).
+
+## Step 2: Choose a Metric
+
+Errors and Transactions each have different metrics for alerting.
+
+MIM NOTE: Ask Adi for a use-case example.
```
Comment (MimiDumpling) [inline src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx:17]: @adhiraj will you provide a use case for this section?

File: src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+Decide how often you want your metric evaluated. Your choices are 1 minute, 5 minutes, 10 minutes, 15 minutes, 30 minutes, 1 hour, 2 hours, and 4 hours.
+
+<Note>
+<markdown>
+Triggers are evaluated every minute regardless of this value.
+</markdown>
+</Note>
+
+MIMI NOTE: Ask Adi for a use-case example.
```
Comment (MimiDumpling) [inline src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx:45]: @adhiraj will you provide a use case for this section?

File: src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+- Send an email (to a member or team). If sent to a member, the member's personal project alert opt-out settings are overridden.
+- Send a [Slack](/product/integrations/slack/) notification.
+- Trigger a [PagerDuty](/product/integrations/pagerduty/) incident.
+- Send a request to a webhook (via [internal integrations](/product/integrations/integration-platform/#internal-integrations)).
+
+## Step 7: Give the Alert Rule a Name
+
+Manage your alert rule by giving it a descriptive name. A descriptive alert rule name specifies the team affected and the topic of the alert. For example, "Frontend Latency," "Backend Failure Rate," or "Billing Apdex."
+
+MIMI NOTE: Ask Adi for a use case
```
Comment (MimiDumpling) [inline src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx:70]: @adhiraj will you provide a use case for this section?

File: src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+## Alert Stream
+
+The alert stream's default view shows unresolved alerts. Use auto-resolve (the default) in a metric alert if you mostly care about a sustained (not transient) change in your metric. For example, if there's an occasional latency spike in an endpoint caused by a single user, that's just noise. You can set the alert to auto-resolve.
+
+On the other hand, if you get an alert for a spike in errors, that could indicate a real incident even if it subsides. You probably want the alert to stick around and show up in your default view. Therefore, you should unset the auto-resolve option.
+
+![A list of unresolved alerts in the Alert stream.](alert-stream2.png)
+
+MIMI NOTE: Ask Adi if there are best practices for Alert Builder and Alert Details.
```
Comment (MimiDumpling) [inline src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx:84]: @adhiraj will you provide a use case for Alert Builder and Alert Details?

File: src/docs/product/sentry-basics/guides/alert-notifications/notifications.mdx [hunk 6]
```diff
@@ -0,0 +1,39 @@
+---
+title: Notifications
+sidebar_order: 3
+description: "Learn more about creating and managing Notifications."
+---
+
+Manage [your notifications](https://sentry.io/settings/account/notifications/) by navigating to **Settings > Accounts > Notifications**.
+
+MIMI NOTE: Ask Adi about use-cases for each scenario.
```
Comment (MimiDumpling) [inline src/docs/product/sentry-basics/guides/alert-notifications/notifications.mdx:9]: @adhiraj will you provide a use cases for Notifications?

File: src/docs/product/sentry-basics/guides/alert-notifications/index.mdx [hunk 7]
```diff
@@ -6,38 +6,39 @@ redirect_from:
 description: "Learn more about how to both understand and manage the alerts and notifications that give real-time visibility into errors happening in your code."
 ---
 
-Sentry’s alert notifications give real-time visibility into errors happening in your code and the impact they are having on your users regardless of where your code is deployed. They explain why you’re being notified and uncover meaningful and customizable information about the error and its context. In addition, Sentry will also notify your project team members about various workflow activities.
+Sentry’s alert notifications give real-time visibility into errors happening in your code and the impact on your users regardless of where your code is deployed. Alerts explain why you’re being notified and uncover meaningful and customizable information about the error and its context. In addition, Sentry will notify your project team members about various workflow activities.
```
Comment (PeloWriter) [inline src/docs/product/sentry-basics/guides/alert-notifications/index.mdx:9]: Do want to separate alerts from notifications?
```suggestion
Both Sentry's alerts and notifications provide real-time visibility into errors happening in your code and the impact on your users regardless of where your code is deployed. Alerts uncover meaningful and customizable information about an error and its context. In addition, Sentry send notifications to your project team members regarding various workflow activities.
```

File: src/docs/product/sentry-basics/guides/alert-notifications/index.mdx [hunk 8]
```diff
@@ -6,38 +6,39 @@ redirect_from:
...
 
-## **Alert Notifications**
+## Alerts
 
-When you create a new project in Sentry, a **default alert rule** populates within the project's Alerts view. The default alert notifies **all** project team members any time an error is **first** encountered (for example, when a new **issue** is created) in this project. This notification is sent via email and any other _Legacy Integration_ services you might have enabled.
+When you create a new project in Sentry, you can choose from a few default alerts or create one later. To create an alert, navigate to **Alerts > New Alert Rule** and choose between a [Metric Alert](/product/sentry-basics/guides/alert-notifications/metric-alerts/) or an [Issue Alert](/product/sentry-basics/guides/alert-notifications/issue-alerts/).
```
Comment (PeloWriter) [inline src/docs/product/sentry-basics/guides/alert-notifications/index.mdx:13]:
```suggestion
When you create a new project in Sentry, you can choose from among a few default alerts, or choose to create alerts at a later time. To create an alert, navigate to **Alerts > New Alert Rule**, then select whether to create a [Metric Alert](/product/sentry-basics/guides/alert-notifications/metric-alerts/) or an [Issue Alert](/product/sentry-basics/guides/alert-notifications/issue-alerts/).
```

File: src/docs/product/sentry-basics/guides/alert-notifications/index.mdx [hunk 9]
```diff
@@ -6,38 +6,39 @@ redirect_from:
...
 
-![Default Alert](alerts_notifications_01.png)
+![Choice between Metric Alert or Issue Alert](create-new-alert-rule.png)
 
-Getting notified **once** when an error is first seen is generally useful for catching new errors in real-time, but will not notify you about subsequent instances of the same errors. This means that with only the default alert rule in place, you might be losing visibility into critical issues building up in your code.
+Metric alerts are usually a higher signal than issue alerts because you know upfront what you're monitoring. In contrast, issue alerts power a triage workflow — you cast a wide net for potential problems and review them as they happen.
```
Comment (PeloWriter) [inline src/docs/product/sentry-basics/guides/alert-notifications/index.mdx:17]: I'm not sure what we mean by "higher signal" here - it comes across as jargon - can we simplify?

File: src/docs/product/sentry-basics/guides/alert-notifications/index.mdx [hunk 10]
```diff
@@ -6,38 +6,39 @@ redirect_from:
...
 
-For tighter control over the alerts you're receiving, create **custom alert rules** in your project settings.
+Choosing a default alert, such as getting notified only **once** when an error is first seen, is generally useful for catching new errors in real-time, but will not notify you about subsequent instances of the same errors. This means that with only the default alert rule in place, you might be losing visibility into critical issues building up in your code.
```
Comment (PeloWriter) [inline src/docs/product/sentry-basics/guides/alert-notifications/index.mdx:19]:
```suggestion
Choosing a default alert, such as getting notified only **once** when an error is first seen, is generally useful for catching new errors in real-time, but means Sentry will not alert you about subsequent instances of the same error. This means that with only the default alert rule in place, you might be losing visibility into critical issues building up in your code.
```

File: src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+- `count()`
+- `p50()`
+- `p75()`
+- `p95()`
+- `p99()`
+- `p100()`
+
+## 3: Choose a Time Window
+
+Decide how often you want your metric evaluated. Your choices are 1 minute, 5 minutes, 10 minutes, 15 minutes, 30 minutes, 1 hour, 2 hours, and 4 hours.
```
Comment (PeloWriter) [inline src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx:37]: To make this more maintainable, perhaps:
```suggestion
Decide how often you want your metric evaluated. Your choices range between one minute and four hours.
```

File: src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+</markdown>
+</Note>
+
+MIMI NOTE: Ask Adi for a use-case example.
+
+## 4: Choose an Environment
+
+Specify which [environment(s)](/platforms/javascript/configuration/environments/) will use this particular alert rule. `Environment` is a Sentry supported tag that you can (and should) add to your SDK. Generally, the [tag](/platforms/javascript/enriching-events/tags/) accepts any value but is targeted to refer to your code deployments' naming convention such as _Development_, _Testing_, _Staging_, or _Production_. The environment drop-down list is populated with the environment tag values available in your [Event Stream](/product/accounts/quotas/manage-event-stream-guide/).
+
+Each environment necessitates different levels of urgency and workflow. The urgency and workflows you apply to Production alerts might differ from those you apply to alerts originating from your QA environment.
```
Comment (PeloWriter) [inline src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx:51]:
```suggestion
Each environment typically necessitates different levels of urgency and workflow. The urgency and workflows you apply to Production alerts might differ from those you apply to alerts originating from your QA environment, for example.
```

File: src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+
+## 5: Set a Threshold
+
+Thresholds are numerical values that help define an alert trigger. The Warning trigger's threshold **must be breached** before the Critical trigger. Triggers are evaluated approximately every minute from the highest severity to lowest.
```
Comment (PeloWriter) [inline src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx:55]:
```suggestion
Thresholds are numerical values that help define an alert trigger. The Warning trigger's threshold **must be breached** before the Critical trigger. Triggers are evaluated approximately every minute from the highest to lowest severity.
```

File: src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+
+## 6: Set an Action
+
+Actions define how you and your team will be alerted. Possible actions are:
```
Comment (PeloWriter) [inline src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx:59]:
```suggestion
Actions define how you and your team will be alerted. Possible actions include:
```

**Review by PeloWriter — APPROVED** [review]
This is a good update. We can always iterate on this content to add use cases in a later PR, @adhiraj , if that's better for the team's schedule.

**Pull request merged**